litellm/proxy/_experimental/mcp_server/auth/user_api_key_auth_mcp.py

```diff
@@ -55,17 +55,25 @@ async def process_mcp_request(scope: Scope) -> Tuple[UserAPIKeyAuth, Optional[st
         mcp_servers_header = headers.get(MCPRequestHandler.LITELLM_MCP_SERVERS_HEADER_NAME)
         verbose_logger.debug(f"Raw MCP servers header: {mcp_servers_header}")
         mcp_servers = None
-        if mcp_servers_header:
+        if mcp_servers_header is not None:  # Changed from 'if mcp_servers_header:' to handle empty strings
             try:
-                mcp_servers = json.loads(mcp_servers_header)
+                # First try to parse as JSON array for backward compatibility
+                try:
```

Comment:
this is not launched, you don't need to maintain backwards compatibility. Just accept csv

Reply:
What if a user passes mcp = ["mcp_1",     "mcp2", "mcp3"] 

do you have any validation to show them what they entered was wrong

Reply:
adding validation won't really work, because the interface on the user end is not owned by us, we can respond in api, but we cannot be sure that the client user is using would pick up our responses.